circuitbreakerpanel/circuitbreakerpanel-java-mvn/circuitbreakerpanel-impl/src/main/resources/application.conf

```diff
@@ -0,0 +1,13 @@
+play.modules.enabled += com.lightbend.lagom.recipes.cbpanel.impl.DemoModule
+
+lagom.circuit-breaker {
+
+  breakerA {
+    max-failures = 3
+    reset-timeout = 20s
+    exception-whitelist = ["java.util.NoSuchElementException"]
+  }
+
+  default.call-timeout = 5s
+
+}
```

Comment:
This is a key piece of the recipe, please make sure there's comments and/or cross-links to docs.